WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/ApplicationLogViewController.swift

```diff
@@ -212,20 +212,14 @@ private extension ApplicationLogViewController {
     func logFileWasPressed(in row: Int) {
         let logFileInfo = logFiles[row]
 
-        let identifier = ApplicationLogDetailViewController.classNameWithoutNamespaces
-        guard let appLogDetailVC = UIStoryboard.dashboard.instantiateViewController(identifier: identifier) as? ApplicationLogDetailViewController else {
-            DDLogError("Error: attempted to instantiate ApplicationLogDetailViewController. Instantiation failed.")
-            return
-        }
```

Comment:
⚠️  Are we missing a storyboard/nib to remove?

Reply:
Good catch, removed in 3d5621bbb